### PR TITLE
test: Add test case for LogMessage #1331

### LINKED ISSUE #1331: Add test case for LogMessage
Code classes that need to develop unit tests:

- `cn.hippo4j.common.toolkit.logtracing.LogMessage`

Please refer to the manual for the first contribution: [guide](https://hippo4j.cn/community/contributor-guide)

Unit test development refers to the following two examples:

- `cn.hippo4j.core.executor.DynamicThreadPoolExecutorTest`
- `cn.hippo4j.core.executor.ExtensibleThreadPoolExecutorTest`

### PR DESCRIPTION
**Pull request by annippolito**
Fixes #1331

Changes proposed in this pull request:
- Add test case for LogMessage

> Check mailbox configuration when submitting. [Contributor Guide](https://hippo4j.cn/community/contributor-guide)

### CODE REVIEW

File: infra/common/src/test/java/cn/hippo4j/common/toolkit/logtracing/LogMessageTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+    private LogMessage logMessage;
+
+    @BeforeEach
+    public void init() {
+        logMessage = LogMessage.getInstance();
+    }
+
+    @Test
+    void getInstance_shouldReturnANewLogMessageInstance() {
```
Comment (magestacks) [inline infra/common/src/test/java/cn/hippo4j/common/toolkit/logtracing/LogMessageTest.java]: Methods should be camelCased

Comment (magestacks) [reply infra/common/src/test/java/cn/hippo4j/common/toolkit/logtracing/LogMessageTest.java]: The method name should start with test, plus the specific method name

Comment (magestacks) [reply infra/common/src/test/java/cn/hippo4j/common/toolkit/logtracing/LogMessageTest.java]: https://github.com/opengoofy/hippo4j/pull/1340

Comment (annippolito) [reply infra/common/src/test/java/cn/hippo4j/common/toolkit/logtracing/LogMessageTest.java]: Thanks, I will follow the project conventions, so will change it in a bit :-)

But I just want to explain what I think about it: 
_Unit test methods don't need to be called by our code or represents actual requests. They have the freedom to tell a story, and spacing between words helps with that. While spaces aren't allowed in method names, using underscores comes close._

I mean for me would be even better to write the **test** method in this way:
```
void getInstance_should_return_a_new_log_message_instance()
 ```

Comment (magestacks) [reply infra/common/src/test/java/cn/hippo4j/common/toolkit/logtracing/LogMessageTest.java]: From the point of view of Java method naming, I recommend you to use camel case naming.

Comment (annippolito) [reply infra/common/src/test/java/cn/hippo4j/common/toolkit/logtracing/LogMessageTest.java]: just pushed thanks :-)

**Review by magestacks — APPROVED** [review]

**Pull request merged**